### DISCUSSION TITLE
Issue With Deleting Carts

### DISCUSSION THREAD

**Discussion by ElBarbs** [swellstores/community#403] [Help] [answered]
Hi,

I am currently unable to delete any cart from my store, using the Backend API or directly from the Swell dashboard. That was not an issue before.

Any ideas?

**Comment by ChBernat** [comment]
I also have this issue. The cart removal API just stopped working.

**Comment by sags95** [comment]
@ElBarbs We've discovered an issue related to deleting carts and will be releasing a fix shortly. I'll be sure to follow up here once it's been deployed to production.

**Comment by sags95** [comment, marked as answer]
Fix has been deployed, this should be resolved.
